object-store/1.1/modules/ROOT/pages/object-store-connector-reference.adoc

```diff
@@ -345,6 +346,79 @@ If `Expiration Interval` is set to a value lower than or equal to zero, then the
 If not provided, the Mule runtime's default ObjectStoreManager is used. Setting this parameter is only necessary when you want to use a non-default Object Store implementation. For example, if you want to have a store which is backed by Redis or JDBC. |  |
 |===
 
+
+
+[[objectStorev2]]
+=== Object Store (Object Store v2)
+
+
+NOTE: These settings apply only to xref:object-store::index.adoc[Object Store v2].
+
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Persistent a| Boolean | Whether the store is persistent or transient. | true |
+| Max Entries a| | *Not configurable for Object Store v2* 
+
+The maximum number of entries is unlimited.|  |
+| Entry Ttl a| Number a| The entry timeout.
+
+Ignored if the value of `expirationInterval` is less than or equal to `0`.
+
+* Minimum value: 1 second
+* Maximum value: 30 days
+
+If the field contains: 
+
+* No value
++
+** Mule versions 4.2.1 and later: Rolling TTL
+** Mule versions earlier than 4.2.1: Static TTL of 30 days by default
+
+* Less than or equal to `0`
++
+TTL is static and set to the maximum value (2592000 seconds/30 days).
+* Positive number
++
+TTL set to the specified value (between 1 and 2592000 seconds)
+
+*Include link to TTL in FAQ and/or TTL task*
+| 2592000 seconds (30 days)  |
+| Entry Ttl Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS a| The unit for the entry's time to live (`entryTtl`) | SECONDS |
+| Expiration Interval a| Positive number a| How frequently the expiration thread should run.
+
+Set `expirationInterval` to a positive value that less than the value of `entryTtl`.
+
+If `expirationInterval` is set to a value less than or equal to zero, `entryTtl` is ignored. 
+
+The `expirationInterval` field requires that the `entryTtl` field is set;
+otherwise, it is ignored.
+| 1 minute |
+| Expiration Interval Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS a| The expiration interval (`expirationInterval`). | MINUTES |
+| Config Ref a| <<ObjectStoreConnector>> a| A reference to an `os:config` element used to declare this Object Store.
```

Comment:
GLOBALLY be sure that we are making the distinction between object store and Object Store clear and consistent.